From 6122ce5d150643e7ce22d82cb1489750e2638549 Mon Sep 17 00:00:00 2001
From: Dmitry Kazakov <dimula73@gmail.com>
Date: Thu, 17 Apr 2025 21:01:49 +0200
Subject: [PATCH] Fix VP9 hw-accelerated encoder with libvpl

MFX_CODEC_VP9 can be provided by libvpl as well.
---
 meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meson.build b/meson.build
index 7714bcfed7..c9897fdbb4 100644
--- a/meson.build
+++ b/meson.build
@@ -2405,6 +2405,7 @@ elif conf.get('libmfx') == 1
   conf.set10('MFX_CODEC_VP9'.to_lower(), cc.has_header_symbol('mfxstructures.h', 'MFX_CODEC_VP9'))
   warning('libmfx is deprecated. Please run meson setup with -Dlibmfx=disabled to use libvpl instead.')
 elif conf.get('libvpl') == 1
+  conf.set10('MFX_CODEC_VP9'.to_lower(), cc.has_header_symbol('vpl/mfxstructures.h', 'MFX_CODEC_VP9'))
   conf.set10('struct_mfxConfigInterface'.to_lower(), cc.has_header_symbol('vpl/mfxvideo.h', 'struct mfxConfigInterface', prefix: '#include <vpl/mfxdefs.h>'))
 endif
 
-- 
2.25.1
